src/shapes/audio_token.class.js

```diff
@@ -167,20 +177,35 @@
N/H0JMDehfAS8GhoP7ChKLEuEOYPzGUpxmbARhc29nYGCd9v//53AGBnZNBoa/1////739//+/yxgYmG8xMBz4BgCdoGC5WKqFNQAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAAuIwAALiMBeKU/dgAAAAd0SU1FB+UEFRIJC0GoNCMAAAAZdEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIEdJTVBXgQ4XAAAIB0lEQVR42u2d3W4jSRXH/6eq3Om2nZ21M0iAuFjlZm9AuyIWT+C5Q1oJMRFPMPsIIHGBuEDaPMLuE6AEIa20d+QJkIN2BRKsBBELyKBlYk/wV6ddVYcLx5Nk/JF2/JG2fX43ybR90VP1y6lzqquqqV6vI8uQYxjvkfOA9oBhrirHVWLsK0aVgBIAKGQbf/2TgaYnnDLh3Gs6tUSnTgF9BVilwJqy3R9ZE0Y5RuA8AsfY6XPNMA6wRVjC2VWOKokmJFrBZ0ygTAijLSOyDmGfj7XHcwVhGJWcwkmco8Oe0XCGtlcYYkaUeESJ/8h4/FQkuV8eq3DUC9TPeoECE22HMMoxColHPvEskjxcnm6gqBOsfsgyqxSlmDhECYso87YlgGLiOZ949AKidqBXJs7ShSFmFGOHvIiyFHEKCXOUWHQDonaolz5ULbUPw8TjacvWiiLLCiIO89OWrYWJXz9hlGOUOrZa6jnetrL4MTGMg1LPcaljq8rxeiS9QeLwds839PWEmvA4OKD5KlLlJNAZzWGYsSu5SmbQQKnU89x1TK1QAwvKbRbSt+QY5bZ9IblKNnObctu+oAUNUXP3r3KMvY493vH4WLoom+x4fLzXsceLyGvmEkZbxl7H1nKM59It2SbHeL7XsTVt+XGEyQ1kaUgVtF5V1F7HNnJzSPMgYbRllDpWKqF1TYY7tvHQSDOzMMoxyl1bE1nWW5py19YektPMJAw5Rrljj2UY2ozhqdyxx7NWT+mFYUapa19IgrtZiXCpa1+A00uTeqZ3t2dRTJilmTePdkDUitLN4aaKMEEymMGVpt1M8glzkLjFCKMc4+2eb8gM7uaigEEfp8hn7vXgSeyqUhFtR+X0JHbVuXKYMHEo9bwMRVtEM1IUT3nCPTHCEDN2Y1+TJtwudmNfoynp6kRhirGDzLdsH4ZxUIzdbMIox1IVbXnVNCkBHitMMXGQqmi7q6bihDJbjYsukUSXrSeaEGVGhCkkXqKLcL2FxU8XhpiRT6SMFoa5jOc3KyZ1NwxJdBHuyhG9EWXeFOYjaSZhmhOvZ3q1ZTztWFn1L9zBA3hZMDQ8auS1H5GVUlqYMCxZNzokhX0+luYRxnHbDarX61CO8Y22DEfC5GHpv0VDXtMgwgROqiNh+rAUuEG1ZAbCyNRLFuFf/XL04nvvg374weDzzz4Fvvh85Cv0818s/F4Cx4iHOcxOn2UZgzCVoSOKHMsyBuFeDOOAHEMZ76U1hHTSeA+VE1+ElOQ8YLQIk10++NHotfKt9fjfPwDe2V/Z7WgPUO+v/6iGff6d9I5wH3GOninluCpNIaRBOa4qYuxLUwhpIMY+2T9/Jee8ZBT+za9HL76zD6r8YPB57ffA389HO/bHP1nK/TigaUhkyS5ffjl6LYxufv/Pv8d/Z1kRBijJIyRhtjxGmkAQYYSlYRhoQvKYbPLuu6PXvvmtu7/HvdUl4UCTkr98dSzHkAlp6BNOFBPOpSmEVBGGcK68plNpCiENXtMptf72TxRlt2M2/6L/9MfRi+US6NvfGXxe/xfQaI58hb77vaXcTztQZJzUSdnl09+OXnvvfeBaGPzhbOwSTSxJGKcA1RdhhLRJrwKUVWKMkA6rFBRrgiWcSXMIU2UhnPFwX9JVjirSJMI0ho5QvV6X41WFexkex6oAINEKsrRXmIS/dgS4fvjoNcEpnEjTCBPK6ROvB8d9vD4fphhb7F7JYYjCKK0donZobiIMAPSMlmFJGDsc9czNUfKvhXGGYBWOpImEO+W0wtHw9Kk7QxIA5K8cnsRSLQk3XIaKujtjIgwA9AKploQ3hqPg7pOAO/9iInQDRdJUAgB0A0VMNFkYAOhIlBGuo0snGH3OOHLFa0IvIIkyW04vIBrOvUwVBgDagZTY2x5d2hPeyjZWGK8JXYkyW5y7jI8uE4UBgHaoZdnDFmIJZ+3wAe98ZCK0Qi3LHraMVqgrPGVwmbrcLg4UYkPPpBm3g9jQsziYvgLz3vWZl6E+dYPdkcIG44DmZajv3XJ0rzBeE15FqixV02ZXRa8iVZ6U6M4kDAAkgZaqacOromTKy81nFuY6GcKVwofSvJvFlcKHrVCn/n76PSZEaObNJ32SlXmbQp9w0sybTzDD4DHTpiTWhEbBHMr8zPpjCWeNgjlkPVumMfMuNq8JjbypSOW03hVRI28qXs+elj5o26MzhGbBlEWa9ZSlWTDl26voli4MAPQN4aJgyjI8rdcwdFEw5b55eME718ZqN5CmIonweiS4FwVTcWa+2ZG5d+J7TbgomEMpubNdOl8UzOFDcpaRYvn2IvC5YMZu7JBPWF42mhE8BpNyrVADC5p3XVzfEqEVGTQjRZIMZyS5jRS1IrMwWRYrzDVJoPGyaMrylPvxiA09e1k05bTT/Y8zJI0hTBx2Y1+Td0qurgpqhaoSL0GUlQgDAMSMouQ2K8lV2qEGL/kZsVn2f4avc5tOwFRMHCIRZ6Gi9AKidqCxiAooE8LcLr//Fxm0A6ZC4pFPvIgzV0RR1AnUykRZWtKbRpxWpPH1W4YuQ0WJwpEszkonSaJwdBkq+votQ61Ir1yWleQwadCWEVmHsM/H2uO5RJ4bSZzCSZyjw57RmHeWdmOEuRPyHCNwHoFj7PR56yosSzi7ylEl0TQ4Sk5na6Fj5oQZuUHHMN4j5wfvTTbMVeW4Sox9xagOX0Go1iBaAINXyHjCKRPOvaZTS3Tq1ODQZKsUWGd7Jez/AcKgXtqNanSbAAAAAElFTkSuQmCC',
```

Comment:
Once again, not sure how fabric usually handles this, but maybe those images should be in files of their own if possible?

Reply:
Done